corehq/apps/export/custom_export_helpers.py

```diff
@@ -292,9 +292,8 @@ def update_multi_select_column(question, col):
         case_cols = filter(lambda col: col["index"] == FORM_CASE_ID_PATH, column_conf)
         if not requires_case:
             for col in case_cols:
-                if col['index'] == FORM_CASE_ID_PATH:
-                    col['tag'], col['show'], col['selected'] = 'deleted', False, False
-                    col['allOptions'] = []
+                col['tag'], col['show'] = 'deleted', col['selected']
```

Comment:
when an export no longer requires cases, we should show it, iff `col['selected']` is `True` else hide it as deleted. we should never modify the value of `selected` since it'll probably be purposeful, but in case it's not the question is highlighted in orange